README.md

```diff
@@ -118,10 +118,10 @@ To install pgai from source on a PostgreSQL server:
    These extensions are automatically added to your PostgreSQL database when you
    [Enable the pgai extension](#enable-the-pgai-extension-in-your-database).
 
-1. Make this `pgai` extension:
+1. Install the `pgai` PostgreSQL extension:
 
     ```bash
-    make install
+    just ext install
```

Comment:
Can we add 

"or if you don't have just"
```bash
projects/extension/build.py install
```

Reply:
Good idea. Already updated.